controllers/app.controller.js

```diff
@@ -63,9 +66,22 @@ exports.postCommentById = (req, res, next) => {
 
     return addCommentById(articleId, body)
     .then(({rows}) => {
-        res.status(201).send(rows)
+        res.status(201).send({comment:rows})
     })
     .catch((err) => {
         next(err);
     });
+}
+
+exports.getUsers = (req, res, next) => {
```

Comment:
I strongly advise breaking your models and controllers out to reflect the tables of the database you're referencing - as it is, your code will quickly become unreadable and difficult to navigate. For example usersController, commentsController etc.